spec/support/elasticsearch.rb

```diff
@@ -1,8 +1,7 @@
 require 'task_helpers'
 
-def refresh_es_index(index_name=nil)
-  index_name = index_name ? index_name : Content::ES_INDEX_NAME
-  TaskHelpers::ElasticsearchHelper.refresh_index(index_name)
+def refresh_es_indices()
```

Comment:
We create the functions just to call another function... Please could you share me with this pattern please?)

Reply:
Done